src/lib/transformers/tokend-client.js

```diff
@@ -259,6 +259,11 @@ class TokendClient extends Source.Polling(TokendParser) {
         } catch (e) {
           return callback(e, null);
         }
+
+        if (res.statusCode !== 200) {
+          callback(new Error("ERROR: Received a non 200 status code from tokend"), secret.error.message);
```

Comment:
Poopie, this should probably be something along the lines of "ERROR: Tokend failed to decrypt a secret"

Reply:
I'm not sure you want to pass the error message as the `data` argument to the callback as it'll be thrown away when the error is handled. See https://github.com/rapid7/propsd/pull/262/files#diff-b3189f329f8903f3731284636ec9e9f6R150.

A better option might be to include the error message from Tokend in the `Error` that's being passed as the first argument.

Reply:
All set